base/cvd/mkbootfs/mkbootfs.cpp

```diff
@@ -0,0 +1,457 @@
+
```

Comment:
I see the [original file](https://cs.android.com/android/platform/superproject/main/+/main:system/core/mkbootfs/mkbootfs.cpp;l=1;drc=e7f1d41237f2504a0aeb8bcf62c8954b48f282c5) doesn't have a license, should we add one here?

Reply:
Done